template/osfingermap.yaml

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# vim: ft=yaml
+#
+# Setup variables using grains['osfinger'] based logic.
+# You just need to add the key:values for an `osfinger` that differ
+# from `defaults.yaml` + `os_family.yaml`.
```

Comment:
@alxwr If you're still around, then this will be:

```yaml
`defaults.yaml` + `os_family.yaml` + `osmap.yaml`
```

If not, I'll merge anyway and add that to a future PR.

Reply:
Done :-) Thanks for the review!

Reply:
@alxwr Don't go anywhere!  I've got another point that needs to be resolved.  Give me 2-3 minutes...